### ISSUE TITLE
✅ 🔗  🐞 Bug Report: Nav Menu breakpoint not working after update (Duplicate of #210030 - [ED9978]

### ISSUE THREAD

**Issue by louisho5** [elementor/elementor#21255]
### Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

**Situation**
Updating Elementor Pro from Version 3.9.0 to Version 3.10.3

**Issue**
The Mobile Dropdown Breakpoint for Nav Menu is somehow not working any more.

### Steps to reproduce

1) Elementor Pro Version 3.9.0
2) Create a Nav Menu with breakpoint Tablet Extra
3) Update Elementor Pro to Version 3.10.3
4) Go back to Nav Menu and update the breakpoint values
5) Somehow not working any more. Unless reroll Elementor Pro to 3.9.0

### Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [x] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.54 (Debian)
	MySQL version: mariadb.org binary distribution v10.9.3
	PHP Version: 7.4.30
	PHP Memory Limit: 
	PHP Max Input Vars: 5000
	PHP Max Post Size: 512M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	WP Multisite: No
	Max Upload Size: 512 MB
	Memory limit: 40M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 2
	Debug Mode: Inactive

== Theme ==
	Theme Name: Hello Elementor Child
	Theme URI: https://github.com/elementor/hello-theme/
	Description: Hello Elementor Child is a child theme of Hello Elementor, created by Elementor team
	Author: Elementor Team
	Author URI: https://elementor.com/
	Template: hello-elementor
	Version: 1.0.1
	Text Domain: hello-elementor-child
	License: GNU General Public License v3 or later.
	License URI: https://www.gnu.org/licenses/gpl-3.0.html

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36

== Active Plugins ==
	Advanced Custom Fields PRO
		Version: 6.0.7
		Author: WP Engine

	Classic Editor
		Version: 1.6.2
		Author: WordPress Contributors

	Contact Form 7
		Version: 5.7.3
		Author: Takayuki Miyoshi

	Custom Post Type UI
		Version: 1.13.4
		Author: WebDevStudios

	Elementor
		Version: 3.10.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.9.0
		Author: Elementor.com

	Polylang
		Version: 3.3.1
		Author: WP SYNTEX

	Yoast SEO
		Version: 20.0
		Author: Team Yoast


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active by default
	Additional Custom Breakpoints: Active by default
	Hide native WordPress widgets from search results: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Inactive by default
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Active by default
	Landing Pages: Active by default
	Nested Elements: Inactive by default
	Lazy Load Background Images: Inactive by default
	Save as Default: Active by default
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Inactive by default
	Form Submissions: Active by default
	Scroll Snap: Active by default

				
````

</details>

**Comment by Florianvb93** [comment]
I’m having the same issue. Waiting for a fix.

**Quick fix that only worked for me:**

**Reroll Elementor:**

- Elementor v3.9.0
- Elementor Pro v3.8.2

**Reset Breakpoint (tablet)**

- Open theme builder Header
- Open Elementor site settings
- Go to breakpoints
- Reset to 1023px (or something that you have set up)
- Close Elementor site settings
- Hard refresh page
- Check if breakpoint works again
- Reset to the normal breakpoint value (1024px in my case)
- Check if breakpoint works

**Comment by nicholaszein** [comment]
Hello @louisho5, 

Thank you for submitting your report! 🙏 

### ❌ While reviewing your report, we noticed it was missing some key pieces of information:

- 💾 **System info:** Please include the System Info report as outlined in our [guidelines](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml). 
   - Get your System Info from your WordPress `Dashboard` > `Elementor` > `System info` [learn how to copy it here](https://elementor.com/help/hare-my-system-info/). We can't accept issues without it.

We don't see the full extended System Info report. We need to see everything from it, including the logs.

Without them, we simply can't help. 😕 

### 🧐 Please provide the missing information above so we can troubleshoot your issue faster and better.

Best regards,

Nick

**Comment by louisho5** [comment]
Hi Nick,

Some logs should be missing, please find it below. Thanks.

### System Info

<details>
<summary>System Info</summary>

````txt

					
== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.54 (Debian)
	MySQL version: mariadb.org binary distribution v10.9.3
	PHP Version: 7.4.30
	PHP Memory Limit: 
	PHP Max Input Vars: 5000
	PHP Max Post Size: 512M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	WP Multisite: No
	Max Upload Size: 512 MB
	Memory limit: 40M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 2
	Debug Mode: Inactive

== Theme ==
	Theme Name: Hello Elementor Child
	Theme URI: https://github.com/elementor/hello-theme/
	Description: Hello Elementor Child is a child theme of Hello Elementor, created by Elementor team
	Author: Elementor Team
	Author URI: https://elementor.com/
	Template: hello-elementor
	Version: 1.0.1
	Text Domain: hello-elementor-child
	License: GNU General Public License v3 or later.
	License URI: https://www.gnu.org/licenses/gpl-3.0.html

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36

== Active Plugins ==
	Advanced Custom Fields PRO
		Version: 6.0.7
		Author: WP Engine

	Classic Editor
		Version: 1.6.2
		Author: WordPress Contributors

	Contact Form 7
		Version: 5.7.3
		Author: Takayuki Miyoshi

	Custom Post Type UI
		Version: 1.13.4
		Author: WebDevStudios

	Elementor
		Version: 3.10.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.9.0
		Author: Elementor.com

	Polylang
		Version: 3.3.1
		Author: WP SYNTEX

	Yoast SEO
		Version: 20.0
		Author: Team Yoast


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active by default
	Additional Custom Breakpoints: Active by default
	Hide native WordPress widgets from search results: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Inactive by default
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Active by default
	Landing Pages: Active by default
	Nested Elements: Inactive by default
	Lazy Load Background Images: Inactive by default
	Save as Default: Active by default
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Inactive by default
	Form Submissions: Active by default
	Scroll Snap: Active by default


== Log ==
	
Log: showing 20 of 422023-01-09 07:09:24 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.9.1',
  'to' => '3.9.2',
)]
2023-01-09 07:09:58 [info] elementor-pro::elementor_pro_updater Started 
2023-01-09 07:09:58 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-01-09 07:09:58 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-01-09 07:09:58 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.9.1',
  'to' => '3.9.2',
)]
2023-01-17 09:31:51 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.9.2',
  'to' => '3.10.0',
)]
2023-01-17 09:33:04 [info] elementor::elementor_updater Started 
2023-01-17 09:33:04 [info] Elementor/Upgrades - _on_each_version Start  
2023-01-17 09:33:04 [info] Elementor/Upgrades - _on_each_version Finished 
2023-01-17 09:33:04 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.9.2',
  'to' => '3.10.0',
)]
2023-01-30 14:09:20 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.10.0',
  'to' => '3.10.2',
)]
2023-01-30 14:09:20 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.9.2',
  'to' => '3.10.3',
)]
2023-01-30 14:09:39 [info] elementor::elementor_updater Started 
2023-01-30 14:09:39 [info] Elementor/Upgrades - _on_each_version Start  
2023-01-30 14:09:39 [info] Elementor/Upgrades - _on_each_version Finished 
2023-01-30 14:09:39 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.10.0',
  'to' => '3.10.2',
)]
2023-01-30 14:09:41 [info] elementor-pro::elementor_pro_updater Started 
2023-01-30 14:09:41 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-01-30 14:09:41 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-01-30 14:09:41 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.9.2',
  'to' => '3.10.3',
)]

PHP: showing 1 of 1PHP: 2022-12-22 16:09:26 [notice X 1][/var/www/html/wp-content/plugins/elementor-pro/modules/nav-menu/widgets/nav-menu.php::1490] Trying to access array offset on value of type null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]



== Elementor - Compatibility Tag ==

	Elementor Pro: Compatibility not specified

== Elementor Pro - Compatibility Tag ==
	

				
````

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro **[v3.11.5](https://elementor.com/pro/changelog/)**! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
